Source/UserManagement/Domain/DataCollectors/Commands/AddDataCollector.cs

```diff
@@ -4,23 +4,24 @@
  *--------------------------------------------------------------------------------------------*/
 
 using System;
+using System.Collections.Generic;
 using Concepts;
 using doLittle.Commands;
 
-namespace Domain
+namespace Domain.DataCollectors.Commands
 {
     public class AddDataCollector : ICommand
     {
-        //TODO: Update these properties to reflect what is needed for event. Remove PhoneNumber
-        public Guid Id { get; set; }
+        public Guid DataCollectorId { get; set; }
+
         public string FullName { get; set; }
         public string DisplayName { get; set; }
         public int YearOfBirth { get; set; }
-        public Sex Sex { get; set; } //Do we need Transgender / Other?
+        public Sex Sex { get; set; }
         public Guid NationalSociety { get; set; }
         public Language PreferredLanguage { get; set; }
         public Location GpsLocation { get; set; }
-        public string MobilePhoneNumber { get; set; }
+        public List<string> MobilePhoneNumbers { get; set; }
```

Comment:
I'd use string[]

Reply:
Also, its called PhoneNumber in the events (and Origin by @einari in TextMessage and reports).